RecoEcal/EgammaCoreTools/src/EcalClusterTools.cc

```diff
@@ -1307,8 +1307,11 @@ float EcalClusterTools::getNormedIX(const DetId& id)
     if( id.subdetId()==EcalEndcap ) ix = EEDetId(id).ix();
     if( id.subdetId()==EcalShashlik ) ix = EKDetId(id).ix();
     if( ix != 0) {
-      int iXNorm  = ix - 50;
-      if(iXNorm<=0) iXNorm--;
+      float iXNorm; 
+      //in shashlik ix and iy run from 1 to 210
+      if( id.subdetId()==EcalShashlik ) iXNorm  = ix - 105.5;
+      else iXNorm = ix - 50.5;
+      //      if(iXNorm<=0) iXNorm--;
```

Comment:
Yes,I'm almost 100% sure it's the right thing to do. Asked Sasha (and other EE guys) before doing this pull request and they confirmed. If you see the method was already returning a float and not an int.
Anyway it makes no visible difference in the end.